server/stats/achievement_service.py

```diff
@@ -95,7 +95,11 @@ def __init__(self, api_accessor: ApiAccessor):
         Else, it returns None
         """
         self._logger.debug("Updating %d achievements", len(queue))
-        response, content = await self.api_accessor.update_achievements(queue, player_id)
+        try:
```

Comment:
Wouldn't it be better to do the error handling inside of `APIAccessor` instead of achievements service? That way there the error handling won't need to be duplicated everywhere where api_accessor is used

Reply:
We return a response / content pair and don't have a meaningful return value if we get an exception. If we return None, we have to check it everywhere anyway. We can hack in some error-like response/contant pair, but that's ugly and *still* needs manual checking.

Reply:
Ah, yea that's true.

Reply:
:cactus: